### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by sivakaruna**
# Description

This PR includes the following proposed change(s):

- https://github.com/bcgov/jag-bcsc-auth-service/issues/13

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # jag-bcsc-auth-service
 BCSC Auth service (eCRC microservice) by Ministry of Justice
 
+BCSC Auth service API utilizes OIDC library to integrate with the BC Services Card (BCSC) to verify user identity.
```
Comment (sdevalapurkar-bcgov) [inline README.md]:
```suggestion
BCSC Auth service API utilizes the OIDC library to integrate with the BC Services Card (BCSC) to verify user identity. 
```

Comment (sdevalapurkar-bcgov) [reply README.md]: this is a good concise summary; however, reading into the issue ticket maybe we need to provide more details about what the service is? not too sure what else to mention though :)

**Review by sdevalapurkar-bcgov — APPROVED** [review]
LGTM, one comment though and few nits

File: README.md (hunk 1 above)
Comment (sivakaruna) [reply README.md]: maybe I should provide more info about the different endpoints? let me see what else I can add.

Comment (sdevalapurkar-bcgov) [reply README.md]: sounds like a good idea! and also some additional background on information about dependencies on other services being setup and configured (ie. BC Services Card configuration)

Comment (sivakaruna) [reply README.md]: I have added everything that I can think of

**Review by sdevalapurkar-bcgov — APPROVED** [review]

**Pull request merged**